app/design/frontend/base/default/template/ebanx/checkout/success/spei.phtml

```diff
@@ -0,0 +1,31 @@
+<?php
+$order = $this->getOrder();
+$payment = $this->getPayment();
+$dueDate = $this->helper->getDueDate($order->getEbanxDueDate(), 'dd/MM');
+?>
+<div class="ebanx-details">
+    <p>¡Listo <?= $order->getBillingAddress()->getFirstname() ?>! Tu boleta EBANX de pago en SPEI ha sido generada.</p>
+
+    <p>Enviamos una copia a <strong><?= $order->getBillingAddress()->getEmail() ?></strong>.</p>
+
+    <p>No lo olvides: tu boleta vence el día <strong><?= $dueDate; ?></strong>. Después de esa fecha no será posible
+        realizar el pago.</p>
+
+    <p>¿Dudas? Con gusto te <a href="https://www.ebanx.com/mx/ayuda/pagos/boleta" target="_blank">ayudaremos</a>.</p>
```

Comment:
This is linking to oxxo help, remove the link while we don't have it.
Talk to Annia about a help page for SPEI.